docs/CONFIGURATION.md

```diff
@@ -254,6 +255,14 @@ Flag to enable SAML authentication.
 **Default** - False: *empty*<br>
 **Values** - True: `1`, False: *empty*<br>
 
+## `SAML_SLO_ENABLED`
+
+Flag to enable SAML Single Logout authentication.
+
+**Target** - Front-end (web), Back-end (api)<br>
```

Comment:
I don't think there's anything on the backend that currently looks at SAML_SLO_ENABLED (though I have a comment elsewhere where I think it should)

Reply:
yep, it does now.